Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -2130,11 +2134,17 @@ internal extension Purchases {
         self.offeringsManager.invalidateCachedOfferings(appUserID: self.appUserID)
     }
 
-    /// - Throws: if posting events fails
-    /// - Returns: the number of events posted
     @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    @available(
+        *,
+        deprecated,
+        message: """
+            Paywall events are now automatically flushed with background task support. \
+            Manual flushing is no longer necessary.
+            """
+    )
     func flushPaywallEvents(count: Int) async throws -> Int {
-        return try await self.eventsManager?.flushFeatureEvents(batchSize: count) ?? 0
+        return 0
```

Comment:
Hmm I think it might be ok to deprecate this if we think we shouldn't support it anymore (can't think of use cases where this would be needed), but I wonder if we should still keep the flushing behavior, at least for now? Mostly to not cause a behavior change... it shouldn't really be that important though, so I don't feel strongly about this